ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -933,6 +930,21 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
         return {kind, health, sync, namespace, name};
     }
 
+    private getBreadcrumbs(app: appModels.Application) {
```

Comment:
We don't have many frontend tests, but would you be up for unit-testing this function? Looks like a good candidate for testing.

Reply:
I took a look, didnt find where `ui-test` are run?

Reply:
ok found it! some minor fix (` .nav-bar` --> `.sidebar ` ) to do